tools/database/src/make_snapshot.rs

```diff
@@ -1,11 +1,15 @@
-use near_store::{checkpoint_hot_storage_and_cleanup_columns, Mode, NodeStorage, StoreConfig};
+use near_store::{
+    checkpoint_hot_storage_and_cleanup_columns, DBCol, Mode, NodeStorage, StoreConfig,
+};
 use std::path::{Path, PathBuf};
 
 #[derive(clap::Args)]
 pub(crate) struct MakeSnapshotCommand {
     /// Destination directory.
     #[clap(long)]
     destination: PathBuf,
+    #[clap(long)]
+    flat_state_only: bool,
```

Comment:
Can you add a comment that this is the type of snapshot used for state sync and resharding?